mappings/net/minecraft/block/Blocks.mapping

```diff
@@ -38,6 +38,7 @@ CLASS net/minecraft/class_163 net/minecraft/block/Blocks
 	FIELD field_660 CACTUS Lnet/minecraft/class_169;
 	FIELD field_661 CLAY Lnet/minecraft/class_160;
 	FIELD field_662 AIR Lnet/minecraft/class_160;
+		COMMENT
```

Comment:
What's happening with these?

Reply:
I thought of including Javadoc to constants but then I decided to keep it for later

Reply:
It Isn't getting removed for some reason, I'll keep it until I fix it

Reply:
Alrighty